gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -13,14 +13,9 @@ public class AndroidCommandPluginExtension {
 
     private final Project project
 
-    AndroidCommandPluginExtension(Project project) {
+    AndroidCommandPluginExtension(Project project, def androidHome) {
         this.project = project
-        androidHome = readSdkDirFromLocalProperties() ?: System.env.ANDROID_HOME
-        if (androidHome == null) {
-            throw new IllegalStateException("Couldn't read the SDK directory. If you're running the tests, " +
-                    "make sure you set the ANDROID_HOME env. variable and it points to your Android SDK home. Otherwise, " +
-                    "make sure there's a local.properties in the root of your project with the property sdk.dir pointing to the Android SDK")
-        }
```

Comment:
Might be good to keep some checking here, in case the android plugin API changes.